test/node/node.test.js

```diff
@@ -71,4 +71,14 @@ describe('node flags', () => {
         const { stderr } = run(__dirname, ['--node-args', '--unknown']);
         expect(stderr).toContain('node: bad option:');
     });
+
+    it('throws an error if no values were supplied with --max-old-space-size', () => {
+        const { stderr } = run(__dirname, ['--node-args', '--max-old-space-size']);
+        expect(stderr).toContain('Error: missing value for flag --max-old-space-size of type size_t');
```

Comment:
If by chance node decide to change error, this test would fail. Just extract stderr and stdout and check if one is thruthy and the other one is falsy. I think that would be enough